testapi.pm

```diff
@@ -59,13 +59,15 @@ sub record_soft_failure {
 }
 
 sub assert_screen($;$) {
-    my ($mustmatch, $timeout) = @_;
+    my $mustmatch = shift;
+    my $timeout = shift || $bmwqemu::default_timeout;
```

Comment:
Well, the deleted '0 is a valid timeout' still holds, so you should use `//` instead of `||`. Here and everywhere else where 0 is a valid value.